### ISSUE TITLE
AVT 3 - React Expandable Tile has multiple screenreader issues with VoiceOver or JAWS 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3375]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
Voice Over
Note: This issue may be related to the DAP issue #3374.
<and>
iOS 12.3.1
Safari
VoiceOver
<and>
Windows 10 
FireFox Quantum 68.0
JAWS 18
## Detailed Description
1. Start VO
2. Press Control-Option-Shift-Down Arrow to enter the Web area, Nothing focused and VO does not announce anything.
3. Press Control-Option-Right Arrow, VO announces “Expand button main”.
4. Press Control-Option-Right Arrow, VO announces “Above the fold content”.
5. Press Control-Option-Right Arrow, VO announces “Below the fold content”.

## Expected Result:
1. Press Control-Option-Shift-Down Arrow to enter the Web area, VO should announce “Above the fold text”
2. Press Control-Option-Right Arrow, VO announces “Expand button main”, but VO should announce the button label and the current state using aria-expanded=false. 
3. User should be required to press the button and VO should announce the expanded state using aria-expanded=“true” before the “Below the fold content” gets focus. 
4. Press Control-Option-Right Arrow, VO announces should announce “Below the fold content”.  Note: “Below the fold content” is announced when the Tile is not expanded. See screenshot below:
<img width="664" alt="Screen Shot 2019-07-11 at 4 21 03 PM" src="https://user-images.githubusercontent.com/21676914/61087856-72864900-a3fc-11e9-877b-1e210440c935.png">

**Comment by abbeyhrt** [comment]
@dakahn Hey! Did you reopen this issue because the problems weren't fully addressed in #3785?
